table/index_builder.cc

```diff
@@ -109,7 +138,7 @@ Status PartitionedIndexBuilder::Finish(
     // Finish the next partition index in line and Incomplete() to indicate we
     // expect more calls to Finish
     Entry& entry = entries_.front();
-    auto s = entry.value->Finish(index_blocks);
+    auto s = entry.value->IndexBuilder::Finish(index_blocks);
```

Comment:
would adding `using IndexBuilder::Finish` in `ShortenedIndexBuilder` declaration also solve the problem?